README.md

```diff
@@ -67,7 +67,7 @@ _username = input("Please enter your username:")
 __password = getpass("Please enter your password:")
 
 sedas = SeDASAPI(_username, __password)
-result_optical = sedas.search_optical(wkt, startDate, endDate, maxCloudPercent=50)
+result_optical = sedas.search_optical(wkt, startDate, endDate, "", "", maxCloudPercent=50)

```

Comment:
We should probably undo the change to remove the default on these two parameters, passing blank strings is a bit nasty